src/providers/BackgroundReplacementProvider/docs/BackgroundReplacementProvider.mdx

```diff
@@ -129,6 +149,17 @@ const MyChild = () => {
             ? 'Background Replacement Enabled'
             : 'Enable Background Replacement'}
         </button>
+        <button
+            onClick={async () => 
+              await changeBackgroundReplacementOption(
+                backgroundReplacementOption === 'Blue'
+                  ? 'Beach'
+                  : 'Blue')}
+        >
+          {backgroundReplacementOption === 'Blue'
+            ? 'Blue'
+            : 'Beach'}
```

Comment:
we can use a hard coded hex color string as input, and button label can be "Change background to xx color"